_posts/2018-11-28-backlog-priorities.markdown

```diff
@@ -0,0 +1,32 @@
+Agile Priorities
+================
+
+I have an interest in effective software delivery and I also like watching the videos by Gary Strahan on his 'Development That Pays' channel. When a video popped up on my feed entitled 'From Scrum to Scrumban in 6 steps', I dived in with a mix of inquisitiveness (what is this scrumban thing?) and scepticism (not another agile derivative!). However my inquisitiveness won the day as he went through the 6 steps to transform your scrum-type process.
```

Comment:
Could you hyper link to Gary Strahan and his channel? Or add a picture for context?

Reply:
Added links.